docs/integration/staging_area.md

```diff
@@ -0,0 +1,77 @@
+site-specific processes.
+
+Recall that the Data Transfer System (DTS) orchestrates transfers of data
+between the databases of participating organizations like yours. Transferring
+such data invariably involves moving files from a "source" staging area in one
+organization to a "destination" staging area in another. Here, a _staging area_
+is a file system visible to some file transport mechanism such as
+[Globus](https://www.globus.org/). To allow the DTS to access your data for
+transfer, your organization must establish and maintain such a file staging
+area.
+
+At this time, the DTS relies on [Globus](https://www.globus.org) to move
+files between different databases and organizations. Globus is [widely used in
+the scientific research community](https://www.globus.org/user-stories) by
+universities and DOE national laboratories, and its design reflects its focus on
+helping researchers share and access data. Additionally, Globus can be
+integrated with other file storage and transfer platforms like
+[Amazon S3](https://www.globus.org/connectors/amazon-s3),
+[Google](https://www.globus.org/connectors/google-cloud), and many others.
+This interoperability allows Globus clients to interoperate seamlessly even in
+heterogeneous technological settings.
+
+## Setting up a Staging Area
+
+First, your organization must allocate a file staging area on a filesystem that
+you can expose to Globus. The directory structure of this staging area for your staged
+files isn't important to anyone outside your organization, as long as all of the
+files staged there can be made accessible to Globus (and transitively, to the
+DTS). Recall, though, that the `path` element in the
+[Frictionless DataResource](https://specs.frictionlessdata.io/data-resource/)
+metadata specification for each file should match your staging area's directory
+structure--otherwise, the DTS won't be able to locate your file.
+
+Probably the most important decision you'll make about the staging area is the
+directory in the filesystem that serves as the effective "root" directory for
+the staging area. This "root" directory determines the root path you'll use when
+you create a Globus guest collection that allows the DTS to access its files.
+
+The DTS is a work in progress, and we haven't yet worked out all the details for
+handling private or embargoed datasets. We're interested to hear your opinions
+on this and other topics!
+
+## Exposing Your Staging Area to Globus
+
+If your organization has a [Globus subscription](https://www.globus.org/why-subscribe),
+you'll have an easy time configuring a file staging area that can be integrated
+with the DTS. Globus has [extensive documentation](https://www.globus.org/data-sharing)
+to help you understand how to share your data effectively.
+
+To make your staging area available to the DTS, you'll need to set up a Globus
+guest collection. The following links can help you configure your Globus setup:
```

Comment:
this is great -- will be super handy for orgs that don't have something set up yet.

Are you going to mention your globus affiliate code somewhere?

Reply:
I want a t-shirt before I go full-speed on my marketing!